docs/usage.md

```diff
@@ -46,11 +46,13 @@ The `sample` identifiers have to be the same when you have re-sequenced the same
 
 ```csv title="samplesheet.csv"
 sample,run_accession,instrument_platform,fastq_1,fastq_2,fasta
-2612,run1,ILLUMINA,2612_run1_R1.fq.gz,,
-2612,run2,ILLUMINA,2612_run2_R1.fq.gz,,
-2612,run3,ILLUMINA,2612_run3_R1.fq.gz,2612_run3_R2.fq.gz,
+2612,lane1,ILLUMINA,2612_lane1_R1.fq.gz,ILLUMINA,2612_lane1_R2.fq.gz,
+2612,lane2,ILLUMINA,2612_lane2_R1.fq.gz,ILLUMINA,2612_lane2_R2.fq.gz,
+2612,lane3,ILLUMINA,2612_lane3_R1.fq.gz,,
 ```
 
+Please note that the column name `run_accession` is following the ENA terms and is a single or paired-end set of demultiplexed FASTQs. Given that demultiplexing happens per lane, each sequencing pair from each lane is a 'run' and therefore you get multiple 'runs' per sample, which can span across both lanes and sequencing libraries. 
```

Comment:
```suggestion
::: info
Please note that the column name `run_accession` follows the definition of an ENA 'run'.
A 'run' corresponds to a single or paired-end set of demultiplexed FASTQs. 
Given that demultiplexing of a given library happens per lane, each sequencing pair from each lane is a 'run'.
Therefore, for each sample, you may get multiple 'runs' consisting of _both_ lanes (of the same library) _and_ sequencing libraries. 
Therefore ensure that each `run_accession` ID is unique, even if from the same sample!
:::
```

Reply:
What do you think @sofstam ? I tried to be more explicit

Reply:
🚀 Very detailed!